tests/Diggin/RobotRules/Accepter/TxtGoogleWebmasterTest.php

```diff
@@ -22,7 +22,7 @@ public function testCommentAndUseragent()
 $txt = <<<EOF
 User-agent: Googlebot-Mobile
 Disallow: /path/
-# aa
+
```

Comment:
Note that here modifying the existing test:
I couldn't find specifically where the original rules come from...